pkg/version/version.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"fmt"
+	"runtime"
+
+	sdkVersion "github.com/operator-framework/operator-sdk/version"
+)
+
+var (
+	buildVersion     = "unknown"
+	buildGitRevision = "unknown"
+	buildStatus      = "unknown"
+	buildTag         = "unknown"
+
+	// Info exports the build version information.
+	Info BuildInfo
+)
+
+// BuildInfo describes version information about the binary build.
+type BuildInfo struct {
+	Version     string
+	GitRevision string
+	BuildStatus string
+	GitTag      string
+	GoVersion   string
+	GoArch      string
+	OperatorSDK string
+}
+
+func (b BuildInfo) String() string {
+	return fmt.Sprintf("%#v", b)
+}
+
+func init() {
+	Info = BuildInfo{
+		Version:     buildVersion,
+		GitRevision: buildGitRevision,
+		BuildStatus: buildStatus,
+		GitTag:      buildTag,
```

Comment:
Not sure GitTag is the proper name for this since version-release won't exist as a Git tag

Reply:
I think it just shows up as "unknown" then.

Reply:
@rcernich Apologies, pulled this in from the other PR. There it's using $version-$release. So if we do 15 internal builds and a single external build, then the external build is 1.0.0 and the GitTag is 1.0.0-15 while the actual Git tag is 1.0.0

Reply:
I think we should move this discussion to the other PR :)

Reply:
Sgtm. It shouldn't have been in this one to begin with.